### PR TITLE
Translation 1500 feature deriv prime

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -158,7 +158,7 @@
         "format": "prettier --write '**/*.js' '*.js'",
         "stylelint": "stylelint 'src/**/*.js' --formatter verbose",
         "eslint": "DEBUG=eslint:cli-engine eslint {'src/**/*.{js,jsx,ts,tsx}','./*.js'} --fix",
-        "start": "npm run develop",
+        "start": "NODE_OPTIONS=--max-old-space-size=8192 npm run develop",
```
Comment (hasan-deriv) [inline package.json:161]: Are we changing it?

File: src/features/pages/partners/deriv-prime/hero/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                </Typography.Heading>
+                <Typography.Paragraph align="center" textcolor="white" size="large" pb="12x">
+                    <Localize translate_text="_t_Deep, reliable liquidity with fast execution for professionals, seasoned institutions, start-ups, and more_t_" />
+                </Typography.Paragraph>
+
+                <Button.Primary
+                    textsize="large"
+                    hero
+                    id="dm-nav-deriv-prime-contact"
+                    onClick={() => {
```
Comment (mitra-deriv) [inline src/features/pages/partners/deriv-prime/hero/index.tsx:64]: can we use `<a>` here ? 🤔

File: package.json (hunk 1 above)
Comment (aswathy-deriv) [reply package.json:161]: No i will remove it

File: src/features/pages/partners/deriv-prime/hero/index.tsx (hunk 2 above)
Comment (aswathy-deriv) [reply src/features/pages/partners/deriv-prime/hero/index.tsx:64]: We are using the button primary here

Comment (mitra-deriv) [reply src/features/pages/partners/deriv-prime/hero/index.tsx:64]: @aswathy-deriv Yup you are correct, But I remember while reviewing @hasan-deriv PR [here ](https://github.com/binary-com/deriv-com/pull/5433/files#diff-531a38740c1811b8ae406b4c3a8db62cd8b12fe5ed29d9a9565a4c3c6c29b24fR59-R63), he used the `LinkButton.Primary` which is the `<a>`, anything we faced that we changed to button primary? I'm just trying to learn here and see which one to use in the future 🙏

**Review by habib-deriv — DISMISSED** [review]

**Review by fasihali-deriv — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]

**Review by fasihali-deriv — APPROVED** [review]

**Review by mayuran-deriv — APPROVED** [review]

**Review by fasihali-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**